mevolib/fetch/BioSeqs.py

```diff
@@ -1,44 +1,20 @@
-#-------------------------------------------------------------------------------
-#
-#   MEvoLib  Copyright (C) 2016  J. Alvarez-Jarreta
-#
-#   This program comes with ABSOLUTELY NO WARRANTY; for details type `show w'.
-#   This is free software, and you are welcome to redistribute it under certain
-#   conditions; type `show c' for details.
-#
-#-------------------------------------------------------------------------------
-# File:  BioSeqs.py
-# Last version:  v1.11 (17/Jul/2016)
-# Description:  Definition and implementation of the class 'BioSeqs'.
-#-------------------------------------------------------------------------------
-# Historical report:
-#
-#   DATE:  17/Jul/2016
-#   VERSION:  v1.11
-#   AUTHOR(s):  J. Alvarez-Jarreta
-#   CHANGES:  * Minor bugs fixed.
-#
-#   DATE:  15/Jul/2016
-#   VERSION:  v1.10
-#   AUTHOR(s):  J. Alvarez-Jarreta
-#   CHANGES:  * Fixed an important bug in the BioSeqs.from_seqfile() method:
-#                  the default alphabet assigned from Bio.SeqIO.read() and
-#                  Bio.SeqIO.parse() methods when reading certain file formats
-#                  (e.g. FASTA) raises an exception when writing those sequences
-#                  with the Bio.SeqIO.write() method in a GENBANK file. Now a
-#                  more specific alphabet is assigned to the Bio.Seq objects
-#                  created.
-#              * Fixed a bug in the BioSeqs.write() method: the unexpected
-#                  exceptions were not raising after handling them.
-#              * Fixed some documentation errors.
-#
-#   DATE:  25/Nov/2015
-#   VERSION:  v1.00
-#   AUTHOR(s):  J. Alvarez-Jarreta
-#
-#-------------------------------------------------------------------------------
-
-from __future__ import absolute_import
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Definition and implementation of the class 'BioSeqs'."""
+
+from typing import List, Optional
```

Comment:
```suggestion
from typing import Optional
```
Imports follow alphabetical order, so this should be after `import errno`